core/src/streams/clients.rs

```diff
@@ -253,11 +253,23 @@ impl StreamsClients {
                     chunk,
                 );
 
+                let alias_name = config
+                    .events
+                    .iter()
+                    .find(|n| n.event_name == event_message.event_name)
+                    .and_then(|n| n.alias.clone());
+
+                let event_message = EventMessage {
```

Comment:
wonder if we call this publish_message as the ..event_message.clone() before the let is a bit confusing aka can look like your setting itself to itself

Reply:
`publish_message` is used a few lines down for another value, I like to shadow rust varaiables sometimes to indicate the shadowed variable is no longer needed or relevant but I'm happy to call this another name say `aliased_message`.

Reply:
I've made the approach generic so this is no longer relevant